### PR TITLE
Pr/1135

### PR DESCRIPTION
**Pull request by UziTech**
fix merge conflicts and randomize tests

### CODE REVIEW

**Comment by UziTech** [comment]
I think we can merge https://github.com/markedjs/marked/pull/1135 with this PR

File: test/specs/commonmark/commonmark-spec.js [hunk 1]
```diff
@@ -11,42 +11,43 @@ Messenger.prototype.message = function(spec, expected, actual) {
...
-|Hard line breaks            |32 of 36 |89%       |
-|Soft line breaks            |1 of 2   |50%       |
+|Section                                 |Count      |Percent |
+|:---------------------------------------|:---------:|-------:|
+|Tabs                                    |  7 of  11 |     64%|
+|Thematic breaks                         | 18 of  19 |     95%|
+|ATX headings                            | 14 of  18 |     78%|
+|Setext headings                         | 21 of  26 |     81%|
+|Indented code blocks                    | 11 of  12 |     92%|
+|Fenced code blocks                      | 17 of  28 |     61%|
```
Comment (Feder1co5oave) [inline test/specs/commonmark/commonmark-spec.js:35]: HTML blocks is missing

**Comment by Feder1co5oave** [comment]
Travis seems happy.
Let me know on the comment I made in the changeset, then I'll merge.
Github doesn't let me do an automatic merge "because of conflicts", even though I was able to merge with fastforward locally. That's weird.

File: test/specs/commonmark/commonmark-spec.js (hunk 1 above)
Comment (UziTech) [reply test/specs/commonmark/commonmark-spec.js:35]: This table only shows the sections that are not at 100%

Comment (Feder1co5oave) [reply test/specs/commonmark/commonmark-spec.js:35]: I figured. Gonna merge now.

**Pull request merged**